docs/src/index.md

```diff
@@ -1,81 +1,81 @@
-
```

Comment:
what happened to this file that the diff replaces _all_ lines?

Reply:
yeah I don't know, I was concerned to see that too. 
Any ideas?